bundles/org.openhab.binding.nanoleaf/src/main/resources/OH-INF/thing/lightpanels.xml

```diff
@@ -92,4 +93,18 @@
 		</state>
 	</channel-type>
 
+	<channel-type id="swipe">
+		<kind>trigger</kind>
+		<label>@text/channel-type.nanoleaf.swipe.label</label>
+		<description>@text/channel-type.nanoleaf.swipe.description</description>
+		<event>
+			<options>
+				<option value="UP">pressed</option>
+				<option value="DOWN">released</option>
+				<option value="LEFT">long pressed</option>
+				<option value="RIGHT">long pressed</option>
```

Comment:
```suggestion
				<option value="UP">Up</option>
				<option value="DOWN">Down</option>
				<option value="LEFT">Left</option>
				<option value="RIGHT">Right</option>
```

Reply:
This really makes me mad that I overlooked this and which is why there is a reason why 4-eyes work better than 2...